src/mumble/MainWindow.cpp

```diff
@@ -2073,6 +2074,16 @@ void MainWindow::sendChatbarText(QString qsText) {
 	qteChat->clear();
 }
 
+void MainWindow::sendChatbarPlainText(QString qsText) {
+	// Unify line endings
+	QString fixedNLPlain =
+		qsText.replace(QLatin1String("\r\n"), QLatin1String("\n")).replace(QLatin1String("\r"), QLatin1String("\n"));
+
+	sendChatbarMessage(fixedNLPlain.replace(QLatin1String("\n"), QLatin1String("\n<br/>")));
```

Comment:
Why are you replacing by `\n<br/>` instead of only `</br>`?

Reply:
I think this was an oversight during testing. Removed the remaining `\n`s and fixed an issue with HTML still being parsed.